addons/bestja_frontend_ucw/templates/pages/about_us.xml

```diff
@@ -0,0 +1,187 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
```

Comment:
You should probably change `<data>` to `<data noupdate="1">` for all pages including content that can be changed in Website Builder, so our subsequent updates will not overwrite those changes (but do not use `noupdate="1"` in other places).